experimental/torch_xla2/torch_xla2/ops/jaten.py

```diff
@@ -1485,6 +1485,7 @@ def _scatter_index(dim, index):
 
 
 # aten.scatter_add
+@op(torch.ops.aten.scatter_add_)
```

Comment:
for ops that ends with _ ; those are usually mutation ops (ops that both return the result as well as modify the first input). Let's add an entry here: https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/torch_xla2/ops/jaten.py#L49 instead.

Reply:
Done.